### ISSUE TITLE
A Suggestion About Clearing the Color

### ISSUE THREAD

**Issue by CodeDojoOfficial** [dialex/JCDP#14]
I know it may seem like you'll have to have the user call clear(), but even then, you can help them even further. It is just a small bit of code you can add to help your users that extra step. Here is the code:

```java
Runtime.getRuntime().addShutdownHook(new Thread(new Runnable() {
    
    @Override
    public void run() {
        clear();
    }
    
}));
```

All this code does is say that while the program is exiting, run clear. This way, the terminal won't look like a mess after running the code. This is kind of the opposite of a static block, where the contents of a `static { ... }` block are run as the JVM is starting, not ending. So if you wanted to, you can use that as well!

**Comment by dialex** [comment]
OMG, thank you for your (detailed and instructional) suggestion! :D
Where should I place that code? Can you make a pull request? I'll be more than happy to merge it.

**Comment by CodeDojoOfficial** [comment]
Thank you! Since I posted this, I have come up with a much simpler solution involving the java.lang.AutoCloseable interface. I will explain the code in comments once I fork it and submit a pull request.

**Comment by dialex** [comment]
Fixed by #15
